components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
```

Comment:
Please use `CGRectGet*()` methods to access bounds/frame rect properties.

```objc
CGRectMake(0, CGRectGetHeight(self.view.bounds) - exampleListTableViewHeight,
           CGRectGetWidth(self.view.bounds), exampleListTableViewHeight);
```

Reply:
Good point. I missed this one. Addressed in 808dd2535cf1f193b23024da6c1c20b98092bed9.